operator/jsonnet/config.libsonnet

```diff
@@ -183,7 +197,12 @@ local utils = (import 'github.com/grafana/jsonnet-libs/mixin-utils/utils.libsonn
             utils.selector.re('job', '.+-index-gateway-http'),
           ],
         },
-        rows: dropPanels(super.rows, dropList, function(p) true),
+        rows: [
+          r {
+            panels: mapPanels([replaceLabelFormat('Per Pod Latency (p99)', '__auto', '{{pod}}')], r.panels),
+          }
+          for r in dropPanels(super.rows, dropList, function(p) true)
```

Comment:
Do you feel like we should add a comment here on why we need to replace the Label Format? I'm also fine with the comment living above the `replaceLabelFormat` function